src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.ios.cs

```diff
@@ -109,19 +113,17 @@ public override void Draw(CGRect rect)
 			currentPath.Stroke();
 		}
 
-		void AddPointToPath(CGPoint currentPoint)
-		{
-			currentPath.AddLineTo(currentPoint);
-			SetNeedsDisplay();
-		}
+		void AddPointToPath(CGPoint currentPoint) => currentPath.AddLineTo(currentPoint);
 
 		void LoadPoints()
 		{
 			currentPath.RemoveAllPoints();
 			foreach (var line in Element.Lines)
 			{
-				UpdatePath(line);
```

Comment:
the root cause of the issue. infinite smoothing the path with updating all points. now points are not changed. only visual  picture is smoothed